spec/views/two_factor_authentication/otp_verification/show.html.slim_spec.rb

```diff
@@ -14,8 +14,6 @@
     end
 
     context 'common OTP delivery screen behavior' do
-      it_behaves_like 'an otp form'
```

Comment:
no it does! I meant to move it to another context further down, as only an existing user will see this behavior.